setup.cfg

```diff
@@ -1,10 +1,115 @@
-[versioneer]
-VCS = git
-style = pep440
-versionfile_source = napari/_version.py
-versionfile_build = napari/_version.py
-tag_prefix = v
-parentdir_prefix = napari-
+[metadata]
+name = napari
+url = https://napari.org
+download_url = https://github.com/napari/napari
+license = BSD 3-Clause
+license_file = LICENSE
+description = n-dimensional array viewer in Python
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = napari team
+author_email = napari-steering-council@googlegroups.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: X11 Applications :: Qt
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: C
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Visualization
+    Topic :: Scientific/Engineering :: Information Analysis
+    Topic :: Scientific/Engineering :: Bio-Informatics
+    Topic :: Utilities
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: MacOS
+
+
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+include_package_data = True
+install_requires =
+    cachey>=0.2.1
+    dask[array]>=2.1.0
+    imageio>=2.5.0
+    importlib-metadata>=1.5.0 ; python_version < '3.8'
+    ipykernel>=5.1.1
+    IPython>=7.7.0
+    napari-plugin-engine>=0.1.5
+    napari-svg>=0.1.3
+    numpy>=1.10.0
+    numpydoc>=0.9.2
+    Pillow!=7.1.0,!=7.1.1  # not a direct dependency, but 7.1.0 and 7.1.1 broke imageio
+    psutil>=5.0
+    PyOpenGL>=3.1.0
+    PyYAML>=5.1
+    qtconsole>=4.5.1
+    qtpy>=1.7.0
+    scipy>=1.2.0
+    tifffile>=2020.2.16
+    toolz>=0.10.0
+    vispy>=0.6.4
+    wrapt>=1.11.1
+
+
+# for explanation of %(extra)s syntax see:
+# https://github.com/pypa/setuptools/issues/1260#issuecomment-438187625
+# this syntax may change in the future
+
+[options.extras_require]
+pyside2 = 
+    PySide2>=5.12.3,<5.15.0
+pyqt5 = 
+    PyQt5>=5.12.3,<5.15.0
+# all will be the full "batteries included" extra.
+# though for now, it's just the GUI backend
+all =
+    %(pyqt5)s
+tests = 
```

Comment:
I find including this in the distribution weird since the users won't ever really need these dependencies. If a user can see this popping up in pip's documentation it could be confusing.

Reply:
I could get behind removing that.  Originally, this PR had removed all the the requirements folder, but that one specifically seems like a good candidate to leave in requirements/tests.txt if you like that better

Reply:
ok, @kne42, I went back to keeping `dev`, `docs`, `release`, and `test` in the requirements folder... I agree those should not be included as extras in the distribution.  (and it also makes it easier to install them without installing the package in editable mode).

the one difference I left in is that `requirements.txt` is now:
```
-e ".[all]"
-r requirements/development.txt
```

so it's a one-stop-shop for setting up a dev environment.  that ok?